cannot install with React 17

I believe this is a regression due to #141 changing the react dependency from v^17.0.1 to a peerDependency on v^16.8.0

peerDependency should probably be `"react": "^16.8.0 || ^17.0.1"`


#### Steps to reproduce

```
npx run create-react-app googlemap
cd googlemap
npm install @googlemaps/react-wrapper
```
#### Stack trace
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: googlemap@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from @googlemaps/react-wrapper@1.1.3
npm ERR! node_modules/@googlemaps/react-wrapper
npm ERR!   @googlemaps/react-wrapper@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

Following these steps will guarantee the quickest resolution possible.

Thanks!

--- a/package.json
+++ b/package.json
@@ -46,7 +46,7 @@
     "@testing-library/react": "^12.0.0",
     "@types/google.maps": "^3.45.6",
     "@types/jest": "^27.0.1",
-    "@types/react": "^17.0.2",
+    "@types/react": ">=16.8.0",
     "@types/react-dom": "^17.0.1",
     "@typescript-eslint/eslint-plugin": "^4.16.1",
     "@typescript-eslint/parser": ">=4.16.1",
@@ -55,7 +55,7 @@
     "eslint-plugin-react": "^7.22.0",
     "jest": "^26.6.3",
     "prettier": "^2.0.4",
-    "react": "^16.8.0",
+    "react": ">=16.8.0",
     "react-dom": "^17.0.1",
     "rollup": "^2.40.0",
     "rollup-plugin-filesize": "^9.0.0",
@@ -74,6 +74,6 @@
     "@googlemaps/js-api-loader": "^1.11.1"
   },
   "peerDependencies": {
-    "react": "^16.8.0"
+    "react": ">=16.8.0"
   }
 }

